src/components/header/HeaderCurrency.tsx

```diff
@@ -12,54 +13,50 @@ import {
   HeaderDropdownWrapper,
   HeaderDropdownTitle
 } from './HeaderMenu.style'
-
-export type HeaderCurrencyItem = {
-  label: string
-  value: string
-}
-
-export type HeaderCurrencyItems = HeaderCurrencyItem[]
+import { toHeaderCurrencyLabel } from './util'
 
 type Props = {
   isDesktopView: boolean
-  items: HeaderCurrencyItems
+  pools: PricePools
```

Comment:
No need to push all data of `PricePools[]` into component, `PricePoolAsset[]` should be fine. That will be fixed with another PR.

Reply:
https://github.com/thorchain/asgardex-electron/issues/185